addons/api/addon/models/session.js

```diff
@@ -6,6 +6,8 @@ import { attr } from '@ember-data/model';
 export default class SessionModel extends GeneratedSessionModel {
   // =attributes
 
+  @attr('boolean', { readOnly: true }) acknowledged;
+  @attr('boolean', { readOnly: true }) started_desktop_client;
```

Comment:
Is this an indication that user clicked on 'connect' button and got a successful response back?

Reply:
Essentially yes.  This is a client-side-only property indicating that the session was started _via this client_ as opposed to some other client.  However upon further reflection, these could be `@tracked` properties instead of model attributes.  When changing the values of tracked properties, you'll need to do so directly on the instances @calcaide, rather than via the store push method we currently use.

Reply:
@randallmorey cool! that makes sense. I will check some info about `@tracked` and do such change.

Reply:
These two variables has been converted to `@tracked`.